src/libraries/Common/tests/TestUtilities/System/SecretsHelper.Net.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Net;
+
+namespace System
+{
+    public static partial class SecretsHelper
```

Comment:
If we decide to proceed with this approach, there should be code comments explaining why is it here, i.e.: to obfuscate pieces of code that trigger false positives reported by internal security tool.